x-pack/solutions/observability/plugins/observability_streams_wrapper/kibana.jsonc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "plugin",
+  "id": "@kbn/streams-app-wrapper-plugin",
+  "owner": "@elastic/streams-program-team",
+  "group": "platform",
```

Comment:
Shouldn't that be `"observability, private"`?

Reply:
It should, good catch. I will create a fix